From 557b7053edbfcee6bbf547ba9e50745ba362a5a6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Thomas=20Hellstr=C3=B6m?= <thomas.hellstrom@linux.intel.com>
Date: Fri, 22 Dec 2023 09:49:11 +0100
Subject: [PATCH] nightly.conf: Add the xe repo to drm-tip
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add the xe repo to drm-tip and the dim tools.
For now use the sha1 of the first drm-xe-next pull request for drm-tip,
since that branch tip is currently adapted for our CI testing.

Cc: Rodrigo Vivi <rodrigo.vivi@intel.com>
Cc: Lucas De Marchi <lucas.demarchi@intel.com>
Cc: Oded Gabbay <ogabbay@kernel.org>
Cc: daniel.vetter@ffwll.ch
Cc: Maarten Lankhorst <maarten.lankhorst@linux.intel.com>
Cc: dim-tools@lists.freedesktop.org
Cc: dri-devel@lists.freedesktop.org
Cc: intel-gfx@lists.freedesktop.org
Signed-off-by: Thomas Hellström <thomas.hellstrom@linux.intel.com>
---
 nightly.conf | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/nightly.conf b/nightly.conf
index 24126b61b797..accd3ff2cc39 100644
--- a/nightly.conf
+++ b/nightly.conf
@@ -24,6 +24,10 @@ git://anongit.freedesktop.org/drm-tip
 https://anongit.freedesktop.org/git/drm/drm-tip
 https://anongit.freedesktop.org/git/drm/drm-tip.git
 "
+drm_tip_repos[drm-xe]="
+ssh://git@gitlab.freedesktop.org/drm/xe/kernel.git
+https://gitlab.freedesktop.org/drm/xe/kernel.git
+"
 drm_tip_repos[drm-intel]="
 ssh://git.freedesktop.org/git/drm/drm-intel
 ssh://git.freedesktop.org/git/drm-intel
@@ -65,14 +69,17 @@ drm_tip_config=(
 	"drm			drm-fixes"
 	"drm-misc		drm-misc-fixes"
 	"drm-intel		drm-intel-fixes"
+	"drm-xe			drm-xe-fixes"
 
 	"drm			drm-next"
 	"drm-misc		drm-misc-next-fixes"
 	"drm-intel		drm-intel-next-fixes"
+	"drm-xe			drm-xe-next-fixes"
 
 	"drm-misc		drm-misc-next"
 	"drm-intel		drm-intel-next"
 	"drm-intel		drm-intel-gt-next"
+	"drm-xe			drm-xe-next b6e1b7081768"
 
 	"drm-intel		topic/core-for-CI"
 	"drm-misc		topic/i915-ttm"
-- 
2.42.0

